app/src/pages/organization/organizationUsersPage/organizationUsersListTable/organizationUsersListTable.jsx

```diff
@@ -0,0 +1,205 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { AbsRelTime } from 'components/main/absRelTime';
+import { MeatballMenuIcon, Popover } from '@reportportal/ui-kit';
+import { urlOrganizationAndProjectSelector } from 'controllers/pages';
+import { SORTING_ASC, withSortingURL } from 'controllers/sorting';
+import { DEFAULT_SORT_COLUMN } from 'controllers/members/constants';
+import {
+  DEFAULT_PAGE_SIZE,
+  DEFAULT_PAGINATION,
+  PAGE_KEY,
+  withPagination,
+} from 'controllers/pagination';
+import {
+  prepareActiveOrganizationUsersAction,
+  usersPaginationSelector,
+} from 'controllers/organizations/users';
+import { SORTING_KEY } from 'controllers/organizations/projects';
+import { ADMINISTRATOR } from 'common/constants/accountRoles';
+import { MembersListTable } from '../../common/membersPage/membersListTable';
```

Comment:
Let's sync on this unification with BA just in case.
I suppose the organization users page may have differences with project members page. In this case it would be easier to maintain the more granular reusable parts other than the whole table component.

Reply:
Synced! They basically have the same functionality